packages/app-store/ee/routing-forms/components/SingleForm.tsx

```diff
@@ -192,6 +192,10 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
     defaultValues: form,
```

Comment:
I don't think you still need this line (probably setting it twice now)

Reply:
Yeah, technically, I don't. But I still find it good to give `defaultValues` as it declaratively tells what value does the hookForm have. Makes the code more readable in a way.

Plus, I think SSR won't work. I will be back with testing results.

Reply:
Yeah, I am doing SSR for the pages and removing defaultValues would cause SSR to not work fully.